source/reference/api/start.txt

```diff
@@ -91,6 +91,37 @@ Request Body Parameters
      - Required
      - Name of the destination cluster.
 
+   * - ``buildIndexes``
+     - boolean
+     - Optional
+     - Configures index builds during sync.
+
+       Supported Options:
+
+       - ``beforeDataCopy`` (the default) causes ``mongosync`` to build indexes
+         on the destination cluster. These include both existing indexes and
+         any indexes created during migration on the source cluster.
+
+       - ``never`` causes ``mongosync`` to skip building unnecessary indexes 
+         during sync.  This can improve migration performance, especially with 
+         index heavy workloads.
+
+         .. warning::
+
+            Do **not** manually build indexes while ``mongosync`` is 
+            performing a migration.  Wait until the migration is complete.
```

Comment:
I'd also say "fully committed" instead of complete

Reply:
Fixed